libraries/AP_Camera/AP_RunCam.cpp

```diff
@@ -118,13 +118,24 @@ AP_RunCam::Menu AP_RunCam::_menus[RUNCAM_MAX_DEVICE_TYPES] = {
 	{ 6, { 3, 10, 2, 2, 8 }}, // Runcam 2 4K
 };
 
-AP_RunCam::AP_RunCam()
+const char* AP_RunCam::_models[RUNCAM_MAX_DEVICE_TYPES] = {
```

Comment:
```suggestion
static const char* AP_RunCam::_models[RUNCAM_MAX_DEVICE_TYPES] {
```

Reply:
naughty, naughty - asking me to change to invalid C++. This is already declared static in the header (where it should be).